lmdeploy/pytorch/supported_models.py

```diff
@@ -42,6 +42,12 @@
     Qwen2MoeForCausalLM=True,
     # Dbrx 132B
     DbrxForCausalLM=True,
+    # cogvlm-chat
+    CogVLMForCausalLM=True,
+    # llava
+    LlavaLlamaForCausalLM=False,
+    # deepseekvl
+    MultiModalityCausalLM=False,
```

Comment:
Does it mean other VL model architecture supported by turbomind also needs to be updated to False here?

Reply:
if arch not exists, it would be `False`

Reply:
Previously, users used Pytorch engine to run VL models, they would get `not supported` hint. However, the current branch will lead users to fix errors individually and finally get nothing but failure.

Reply:
it would give a warning message before the error. the same logic as LLM

https://github.com/InternLM/lmdeploy/blob/7ee48ed006f022d9e9f55b7648d6c6fcf6fb49e3/lmdeploy/archs.py#L52